ui/admin/app/routes/scopes/scope/authenticate.js

```diff
@@ -20,13 +20,34 @@ export default class ScopesScopeAuthenticateRoute extends Route {
    * scope and all scopes (for org navigation).
    * @return {Promise} `{scope, scopes, authMethods}`
    */
-  model() {
+  async model() {
     const { id: scope_id } = this.modelFor('scopes.scope');
+    const allScopes = await this.modelFor('scopes').filter(
+      (scope) => scope.isOrg
+    );
+    const scopesIdList = [];
+    const authenticatableAuthMethodsList = [];
+    //iterate through the scopes list and get authMethods for each scope
+    await all(
+      allScopes.map((scope) => {
+        if (scope) {
+          scopesIdList.push(scope.id);
+          authenticatableAuthMethodsList.push(
+            this.store.query('auth-method', {
+              scope_id: scope.id,
+              filter: { authorized_actions: 'authenticate' },
+            })
+          );
+        }
+      })
+    );
+
     return hash({
       scope: this.modelFor('scopes.scope'),
       scopes: this.modelFor('scopes').filter((scope) => scope.isOrg),
       authMethods: this.store.query('auth-method', { scope_id }),
-
+      scopesIdList,
+      authenticatableAuthMethodsList,
```

Comment:
I have it on the controller side because I want to filter/sort the model data after it has been loaded, if this were to exist in the route, then the user would have to refresh the route to get the filtered data.